cfy_manager/main.py

```diff
@@ -321,29 +309,33 @@ def configure(verbose=False,
         config_write_required=True
     )
 
-    logger.notice('Configuring Cloudify Manager...')
+    # logger.notice('Configuring Cloudify Manager...')
+    logger.notice('Configuring desired components...')
     _validate_manager_installed('configure')
-    validate(skip_validations=True)
+    validate(skip_validations=True, components=components)
     set_globals()
 
-    for component in _get_components():
-        component.configure()
+    for component in components:
```

Comment:
better put in a function

Reply:
It's in a few different functions, install, configure, stop and remove. I don't see any value in moving only the loop itself to a function and send a different command accordingly.
Is that what you meant?